index.html

```diff
@@ -3,15 +3,186 @@
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">New</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link campaign">CC Campaign</a>
+                </li>
+            </ul>
+        </div>
+         
+       
+      </nav>
+      <section class="header">
+        <article>
+       <div class="header-1">
+        <h1 class="header-title">Digital Marketing Capstone Project: <br> A User-Friendly Website to Manage and Monitor Smart Strategies</h1>
+        <p class="para-1">This platform enables users to have centralized control over their marketing endeavors and leverage the power of social networks to expand brand awareness and increase conversions.</p>    
+        <strong>2015.10.15(THU) ~ 16(FRI)</strong>
+        <p>@ National Museum of Karea, Art Center Nabi and more.</p>
+       </div>
+      </article>
+      </section>
 
+      <section class="program">
+        <div class="title">Main Program</div>
+        <article>
+        <div class="program-list">
+                <img src="images/program-images/program_icon_01 1.png" alt="icon" width="50px"/>
+                <h3>Lecture</h3>
+                <p class="Program-description">Easily manage and oversee all digital marketing activities</p>
+                </div>
+                <div class="program-list">
+                    <img src="images/program-images/program_icon_02 1.png" alt="icon"  width="50px" />
+                    <h3>CC Exhibition</h3>
+                    <p class="Program-description">Create, edit, and delete posts for social networks</p>
+                </div>
+                <div class="program-list">
+                    <img src="images/program-images/program_icon_03 1.png" alt="icon"  width="50px"/>
+                    <h3>Forum</h3>
+                    <p class="Program-description">Plan and schedule posts for the future, including campaign-specific content.</p>
+                </div>
+                <div class="program-list">
+                    <img src="images/program-images/program_icon_04 1.png" alt="icon" width="50px" />
+                    <h3>Workshop</h3>
+                    <p class="Program-description">Gain insights into the performance of your digital marketing campaigns through analytical tools, reports, and visualizations</p>
+                </div>
+                <div class="program-list">
+                    <img src="images/program-images/program_icon_05 1.png"alt="icon" width="50px" />
+                    <h3>CC <br> Ignite</h3>
+                    <p class="Program-description">Access a comprehensive dashboard that displays key performance indicators and overarching trends.</p>
+                </div>
+               
+            
+        </div>
+    </article>
+        <h2 class="none">Join CC Global Summit 2015</h2>
+        <div class="hole-program">SEE THE WHOLE PROGRAM</div>
+      </section>
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **The pages should look almost identical to the original design.** Small adjustments like text or image changes are acceptable. So Kindly Adjust the Desktop screen size For the heading "See the whole program" it not visible in your home page Kindly Fix it. 
![main](https://github.com/jerymey/Capstone/assets/93598189/aa31783d-86e0-4fda-8356-58d767d089a4)

- The Original Design :- 
![Capture](https://github.com/jerymey/Capstone/assets/93598189/bd25e01b-e90e-4a26-8124-9774e6f20c39)